docs/source/deployorderer/ordererdeploy.md

```diff
@@ -0,0 +1,275 @@
+
+Notice that the `configtxgen` tool is located in the `bin` folder of downloaded Fabric binaries.
+
+Before using `configtxgen`, confirm you have set the `FABRIC_CFG_PATH` environment variable to the path of the directory that contains your local copy of the `configtx.yaml` file. You can verify that are able to use the tool by printing the `configtxgen` help text:
+
+```
+configtxgen --help
+```
+
+#### The `configtx.yaml` file
+
+The `configtx.yaml` file is used to specify the **channel configuration** of new channels. The information that is required to build the channel configuration is specified in a readable and editable form in the `configtx.yaml` file. The `configtxgen` tool uses the channel profiles that are defined in `configtx.yaml` to create the channel configuration block in the [protobuf format](https://developers.google.com/protocol-buffers).
+
+The `configtx.yaml` file is located in the `config` folder alongside the images that you downloaded and contains the following configuration sections that we need to create our new channel:
+
+- **Organizations:** The organizations that can become members of your channel. Each organization has a reference to the cryptographic material that is used to build the [channel MSP](../membership/membership.html).
+- **Orderer:** Which ordering nodes will form the Raft consenter set of the channel.
+- **Policies** Different sections of the file work together to define the channel policies that will govern how organizations interact with the channel and which organizations need to approve channel updates. For the purposes of this tutorial, we will use the defaults that are used by Fabric. For more information about policies, check out [Policies](../policies/policies.html).
+- **Profiles** Each channel profile references information from other sections of the `configtx.yaml` file to build a channel configuration. The profiles are used to create the genesis block of the channel.
+
+The `configtxgen` tool uses `configtx.yaml` file to create the genesis block for the channel. A detailed version of the `configtx.yaml` file is available in the [Fabric sample configuration](https://github.com/hyperledger/fabric/blob/{BRANCH}/sampleconfig/configtx.yaml). Refer to the [Using configtx.yaml to build a channel configuration](../create_channel/create_channel_config.html) tutorial to learn more about the  settings in this file.
+
+#### Generate the genesis block
+
+After you have completed editing the `configtx.yaml`, you can start the process of creating a system channel for the ordering service. Every channel configuration starts with a genesis block. We have already set the environment variables that are required to use the `configtxgen` tool. Run the following command to build the genesis block for `channel1` using the `SampleAppGenesisEtcdRaft` profile:
+
+```
+configtxgen -profile SampleAppGenesisEtcdRaft -outputCreateChannelTx ./channel-artifacts/channel1.tx -channelID channel1
```

Comment:
Hang on, the first sentence of this section said we were going to create the genesis block of the system channel. We've jumped straight to an application channel creation here.

We need to do the steps in this order:
1) generate the system channel genesis block
2) use the system channel genesis block to bootstrap ordering nodes
3) generate application channel create transaction
4) create the application channel on the ordering service

The 'create a channel' tutorial already does these steps (at least steps 1,3,4). Do we want to document the steps twice, or just point to the create a channel tutorial?

Reply:
Good catch.
I grabbed the wrong profile. 
I've updated the instructions to be more specific to the system channel genesis block. 
and used the `TwoOrgsOrdererGenesis:` profile  from the Test network instead.

For steps 3 and 4 I link to those instructions in the create a channel tutorial.